stanford_profile_helper.module

```diff
@@ -29,6 +29,17 @@ function stanford_profile_helper_form_alter(&$form, FormStateInterface $form_sta
     // @link https://www.drupal.org/project/views_bulk_operations/issues/3055770#comment-13116724
     unset($form['header']['views_bulk_operations_bulk_form']['select_all']);
   }
+
+  $embeddable_forms = [
+    'media_library_add_form_embeddable',
+    'media_embeddable_edit_form',
+    'media_embeddable_add_form',
```

Comment:
i was just thinking, shouldn't field permissions module just do these for us?

Reply:
Oh possibly. 

Let me check on that.

Reply:
Yeah, they do.